utils/src/main/scala/za/co/absa/enceladus/utils/config/ConfigUtils.scala

```diff
@@ -67,5 +67,34 @@ object ConfigUtils {
     }
   }
 
+  implicit class ConfigImplicits(config: Config) {
```

Comment:
👍  I like this class. Maybe we should move methods from `ConfigReader` here (not necessarily in this PR).

Reply:
Sure, it makes sense to do something like that in the future. We will need to discuss the specifics, though, because the `ConfigImplicits` uses explicit from the caller's perspective while the `ConfigReader` uses an internally-obtained one.